easybuild/easyconfigs/b/Bazel/Bazel-0.16.0-GCCcore-7.3.0.eb

```diff
@@ -0,0 +1,16 @@
+name = 'Bazel'
+version = '0.16.0'
+
+homepage = 'http://bazel.io/'
+description = """Bazel is a build tool that builds code quickly and reliably. 
+It is used to build the majority of Google's software."""
+
+toolchain = {'name': 'GCCcore', 'version': '7.3.0'}
+
+source_urls = ['https://github.com/bazelbuild/bazel/releases/download/%(version)s']
+sources = ['%(namelower)s-%(version)s-dist.zip']
```

Comment:
@reedts Please include a SHA256 checksum (`--inject-checksums` will do this for you)

Reply:
Oops, my bad. Updated.